### PR TITLE
fix broken nginx ipv6 listeners

### PR DESCRIPTION
**Pull request by gbonfiglio**
This PR fixes both https://github.com/jitsi/jitsi-meet/issues/5209 and https://github.com/jitsi/jitsi-meet/issues/5000

### CODE REVIEW

**Comment by gbonfiglio** [comment]
Yes, I had the same doubt but nginx's default configuration already includes:

```
        listen 80 default_server;
        listen [::]:80 default_server;
```

To be 100% sure I tested with:

1. add the following to sysctl.conf
```
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1
```
2. run sysctl -p to load the new settings
3. restart nginx

And everything keeps working

**Comment by gbonfiglio** [comment]
(do you really need my CLA signed for a 3 lines long PR?)

File: doc/debian/jitsi-meet/jitsi-meet.example [hunk 1]
```diff
@@ -2,6 +2,7 @@ server_names_hash_bucket_size 64;
 
 server {
     listen 80;
+    listen [::]:80;
```
Comment (RalfJung) [inline doc/debian/jitsi-meet/jitsi-meet.example]: At least for me locally, this configuration does not work... the `/etc/nginx/sites-enabled/default` contains
```
server {
        listen   [::]:80 ipv6only=off default_server;
```
and all other vhosts have to match, so I have to do
```
server {
    listen [::]:80;
    server_name meet.hacksaar.de;
    return 301 https://$host$request_uri;
}
```
for Jitsi to work properly.

Comment (RalfJung) [reply doc/debian/jitsi-meet/jitsi-meet.example]: Ah never mind, looks like that's not actually the default but I forgot I changed that. Sorry.

I can confirm that the configuration you are suggesting here works.

**Comment by mightytarzan** [comment]
This change seems to break installation on systems without IPv6 support (e.g., ipv6.disable=1 set on kernel cmdline). Would it be possible to add a corresponding check in the installer?

[jitsi.txt](https://github.com/jitsi/jitsi-meet/files/4460956/jitsi.txt)

**Comment by RalfJung** [comment]
@mightytarzan does 
```
listen   [::]:80 ipv6only=off default_server;
```
work for you?

Or does nginx really not have a way to "listen on all address families" with one uniform configuration?

**Pull request merged**